src/components/Comment/Comment.tsx

```diff
@@ -227,7 +234,15 @@ export const Comment = ({
                 <div className={timestampWrapperStyles}>
                   <Timestamp
                     isoDateTime={comment.isoDateTime}
-                    linkTo={`https://discussion.theguardian.com/comment-permalink/${comment.id}`}
+                    linkTo={joinUrl([
+                      // Remove the discussion-api path from the baseUrl
+                      baseUrl
+                        .split("/")
+                        .filter(path => path !== "discussion-api")
+                        .join("/"),
```

Comment:
Annoyingly, the base url required for building permalink urls does not include the `discussion-api` in the path, even though this is part of the 'base' url. So here I'm removing that part of the path